server/config.go

```diff
@@ -117,11 +110,10 @@ const (
 
 	defaultName                = "pd"
 	defaultDataDir             = "default.pd"
-	defaultHost                = "127.0.0.1"
-	defaultPort                = uint64(1234)
-	defaultClientPort          = uint64(2379)
-	defaultPeerPort            = uint64(2380)
-	defaultHTTPPort            = uint64(9090)
+	defaultAddr                = "127.0.0.1:1234"
+	defaultClientUrls          = "http://127.0.0.1:2379"
+	defaultPeerUrls            = "http://127.0.0.1:2380"
+	defaultHTTPAddr            = "0.0.0.0:9090"
```

Comment:
docker can use it directly, no need pass again. 
btw, this flag will be removed later, so just keep it.